app/src/org/commcare/views/media/AudioController.java

```diff
@@ -28,24 +31,24 @@
      * Button that corresponds to the currentEntity media. Pressing the button
      * should trigger playback control methods here.
      */
-    private AudioButton currentButton;
+    private AudioPlaybackReset currentAudioReset;
```

Comment:
I'm not following why a whole new interface was needed for this. Why can't the `AudioPlaybackButton` still be the variable that is stored here, and have `resetPlaybackState` be a method of that class?